src/config/CMakeLists.txt

```diff
@@ -10,4 +10,9 @@ target_sources(ConfigLib
 target_include_directories(ConfigLib
   PUBLIC
     ${CMAKE_CURRENT_LIST_DIR}
-    )
\ No newline at end of file
+    )
+
+target_link_libraries(ConfigLib
+    PUBLIC 
```

Comment:
![](https://images.news18.com/ibnlive/uploads/2023/01/pak-meme.png)

I'm still looking like this to that PUBLIC haha